src/Native/Bootstrap/main.cpp

```diff
@@ -284,11 +281,10 @@ int main(int argc, char* argv[])
     if (!RtuDllMain(NULL, DLL_PROCESS_ATTACH, NULL))
         return -1;
 
-#if defined(CPPCODEGEN) || defined(PLATFORM_UNIX)
+#if defined(CPPCODEGEN)
```

Comment:
OK. Will do. I was about this myself because it would make the code look slightly better.